src/styles.css

```diff
@@ -235,6 +241,18 @@ nav {
   padding: 0, 8px, 8px, 8px;
 }
 
+.key {
+  margin: 18px 0px 0px 18px;
+  font-size: 20px;
+  padding: 8px;
+  background: #E7C423;
+  width: 100%;
+}
+
+.highlight-card {
```

Comment:
LOVE! Simple but does the job- signifies which recipes the user can cook based on what's in their panty.